content/news/10th-anniversary.md

```diff
@@ -0,0 +1,21 @@
+We are proud to celebrate the 10th anniversary of OpenRA today!
+
+Community member SoScared put together a new fantastic promotional trailer:
+
+<div style="text-align:center" markdown="1">
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/93b7TKooj-M" frameborder="0" allowfullscreen></iframe>
+
+</div>
+
+<hr>
+
+Meanwhile, we are working towards a new playtest. Some core improvements and additions are:
+<ul>
+<li>A new unit control bar that provides mouse and touch accessibility.</li>
+<li>Unit stances that are easier to use and behave more like the original games.</li>
+<li>Overhauled structure hit boxes, with weapons, engineers, etc. accounting for their physical size instead of acting only at their centre.</li>
```

Comment:
Commas made this read weird to me at first. Could use paul's original formatting?

> Overhauled structure hit boxes, with weapons/engineers/etc. accounting for ...